### PR TITLE
Open-smartwatch update

### PR DESCRIPTION
**Pull request by RuffaloLavoisier**
Update Watchface
* Dual Time
* Fitness Tracking

Update Fitness-APP
* Kcalorie Graph
* Steps Graph
* Fitness tracking chart

Please let me know if you need addition/modification for grammar or content !

### CODE REVIEW

File: docs/oswapps/tools.md [hunk 1]
```diff
@@ -27,6 +27,39 @@ If you have no WiFi connection, you can manually set the time and date with the
...
+
+<img src="/assets/apps/tools/app_tool_kcalStats_osw.png" width="256px" style="float:left; margin-right:20px"/>
+You can check the calorie consumption of the latest date. And you can see the average value for each item and the number for the day.
+
+ * Source: [OswAppKcalStats.cpp](https://github.com/Open-Smartwatch/open-smartwatch-os/blob/master/src/apps/tools/OswAppKcalStats.cpp)
+ * Authors: [ruffalolavoisier](https://github.com/RuffaloLavoisier)
+
+<div style="clear: both; margin-bottom:20px"></div>
+
+!!! tip
```
Comment (simonmicro) [inline docs/oswapps/tools.md:50]: I would recommend you to write `### Tip` instead.

File: docs/oswapps/watchfaces.md [hunk 2]
```diff
@@ -9,19 +9,21 @@ The default analog watchface.
 
 Steps are shown in the blue circle, a full rotation is 10800 steps.
 
- * Source: [watchface.cpp](https://github.com/Open-Smartwatch/open-smartwatch-os/blob/master/src/apps/watchfaces/watchface.cpp)
+ * Source: [OswAppWatchface.cpp](https://github.com/Open-Smartwatch/open-smartwatch-os/blob/master/src/apps/watchfaces/OswAppWatchface.cpp)
  * Author: [pauls-3d-things](https://github.com/pauls-3d-things)
 
 <div style="clear: both; margin-bottom:20px"></div>
-
+!!! tip
```
Comment (simonmicro) [inline docs/oswapps/watchfaces.md:16]: ↑

**Review by simonmicro — CHANGES_REQUESTED** [review]
Just that one formatting issue

**Comment by RuffaloLavoisier** [comment]
test

@simonmicro This is a test environment. That's what I wanted. How is it?

**Comment by simonmicro** [comment]
Oh, I did not know that macro - nice!

**Review by simonmicro — APPROVED** [review]

**Comment by RuffaloLavoisier** [comment]
Thank you for merge !

**Pull request merged**